solr/core/src/java/org/apache/solr/packagemanager/PackageUtils.java

```diff
@@ -302,7 +303,7 @@ public static String getCollectionParamsPath(String collection) {
   }
 
   public static void uploadKey(byte[] bytes, String path, Path home) throws IOException {
-    FileStoreAPI.MetaData meta = FileStoreAPI._createJsonMetaData(bytes, null);
+    FileStoreAPI.MetaData meta = ClusterFileStore._createJsonMetaData(bytes, null);
```

Comment:
guessing minimizing change, ubt maybe we can remove `_` ?

Reply:
I've opted out of renaming this '_' methods, as there are several of them and I'm afraid of bloating this PR's diff.